### Задание 1

Начнём с пары вопросов про выборки!

Ваш коллега утверждает, что полученная им выборка точно является репрезентативной, поскольку объем выборки составляет 100 человек. Так ли это?

* Да
* Нет
* **Зависит от способа формирования выборки**

### Задание 2

Соотнесите способы формирования выборки с предложенными примерами.

A – Для того чтобы протестировать курс по введению в статистику и выяснить, насколько хорошо с ним справятся студенты курса по аналитике, случайным образом было выбрано 35 студентов

B – Для исследования взаимосвязи успешности прохождения курса по статистике и бекграунда, студентов разделили на 2 группы программистов и менеджеров. Затем из каждой группы случайным образом извлекли по 30 человек

C – Для проверки знаний студентов с разных онлайн-курсов по анализу данных, случайным образом было выбрано несколько соответствующих курсов, из студентов которых случайным образом было отобрано по 30 человек с разных потоков.

В - **Групповая выборка (cluster sample)**
А - **Стратифицированная выборка (stratified sample)**
С - **Простая случайная выборка (random sample)**

### Задание 3

Студент решил провести исследование отношения к настройкам приватности среди пользователей социальной сети. Для этого он опросил друзей со своей образовательной программы, которые учатся на разных курсах. В презентации он утверждает, что была сформирована случайная выборка, которая является репрезентативной относительно всех пользователей. Так ли это?

* **Это не простая случайная выборка, ведь не каждый член генеральной совокупности мог равновероятно принять участие в тестировании, т.к. студент опросил только своих друзей**

* Такая выборка должна рассматриваться, как стратифицированная выборка, т.к. учащийся разделил всех студентов на страты по N человек и выбрал только одну из них для исследования

* Это может быть простая случайная выборка, т.к. в неё попали пользователи социальной сети, ещё и с разных курсов

### Задание 4

Какие из перечисленных переменных скорее всего являются дискретными?

* пол
* **количество рекламных объявлений**
* названия товаров
* **количество пройденных курсов на Stepik**
* среднее расстояние между пунктами назначения

### Задание 5

Как называются два основных типа переменных в статистике?

* Качественные и номинативные
* Непрерывные и количественные
* Ранговые и номинативные
* **Качественные и количественные**
* Качественные и некачественные

### Задание 6

Соотнесите примеры переменных и их типы.

* Количественная (дискретная) переменная - **количество кликов на баннер**

* Номинативная переменная - **пол студента**

* Ранговая переменная - **рейтинг студентов по успевавемости**

* Количественная (непрерывная) переменная - **среднее время решение этого задания**


### Задание 7

Если успеваемость 100 студентов исследования представлена в ранговой шкале (по убыванию: 1-самый высокий, 2 - ниже и т.д.), тогда верным утверждением будет: 

* У студентов с рангом 1 и 3 такая же разница в успеваемости, как и у студентов с с рангами 3 и 5
* **Студент с рангом 4 имеет более высокие оценки, чем студент с рангом 6, но ниже, чем студент с рангом 3**
* Студент с рангом 4 отстает на два балла от студента с рангом 6
* Ни одно из утверждений верным не является

### Задание 8

Представьте, что последний день месяца был очень жарким, и продажи мороженого резко увеличились в несколько раз по сравнению с предыдущими днями. Как вы считаете, какая из мер центральной тенденции изменится самым значительным образом?

* Медиана
* Мода
* **Среднее**

### Задание 9

В каких случаях вместо среднего значения лучше использовать моду или медиану?

* **Если присутствуют заметные выбросы**
* **Если распределение асимметрично**
* Если распределение является симметричным и унимодальным


### Задание 10

Предположим, вы провели исследование дохода граждан, проживающих в двух разных городах, опросив по 100 человек из каждого города. Распределение заработной платы изображено на графике.

![](https://storage.yandexcloud.net/klms-public/production/learning-content/11/79/613/1762/8060/pasted_image_2.png)

* Среднее значение
* Все три меры дадут приблизительно одинаковый результат
* **Мода (моды)**
* Медиана

### Задание 11

Проанализируйте две гистограммы и выберите верное высказывание.

![](https://storage.yandexcloud.net/klms-public/production/learning-content/11/79/613/1762/8061/1234_QNRNDh9.png)

* Размах примерно одинаковый в обоих случаях
* В первом случае (синее распределение) размах меньше, чем во втором
* **В первом случае (синее распределение) размах больше, чем во втором**


### Задание 12

У какого из распределений наибольшая дисперсия?

* У зеленого распределения
* **У оранжевого распределения**
* У синего распределения


### Задание 13

Создайте список, состоящий из значений от 0 до 99 включительно. Рассчитайте среднеквадратическое отклонение, используя np.std(). При расчете используйте поправку на число степеней свободы. Ответ округлите до тысячных. В качестве десятичного разделителя используйте точку.

Hint: параметр ddof=1

In [1]:
import numpy as np

In [3]:
lst = np.arange(0, 100)
lst

array([ 0,  1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11, 12, 13, 14, 15, 16,
       17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33,
       34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50,
       51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67,
       68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84,
       85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99])

In [8]:
np.std(lst, ddof=1)

29.011491975882016

Ответ: 29.011


### Задание 14

Может ли показатель стандартного отклонения принимать отрицательные значения?

* **Не может, стандартное отклонение всегда неотрицательное**
* Не может, стандартное отклонение всегда равно нулю
* Может, если все значений выборки отрицательные
* Может, если все значения в выборке равны друг другу
* Может, если диспресия выборки принимает неположительное значение